### DISCUSSION TITLE
Own SSL cert and key - where to put?

### DISCUSSION THREAD

**Discussion by tilwen** [dunglas/symfony-docker#359] [Q&A] [answered]
Hi!

I'm trying to get my SSL cert+file configured.

The Caddy documentation says to simply set:
tls /etc/ssl/ssl.cert.pem /etc/ssl/ssl.privkey.pem
I added two COPY lines into the Dockerfile to copy them at build time (actually I should map the files in docker run way right?) it seems it's at least reading them but I get this error:

symfony-docker-caddy-1  | Error: adapting config using caddyfile: server listening on [:80] is HTTP, but attempts to configure TLS connection policies

I understand the error, yeah it doesn't belong to port 80 config, it needs to go into the 443/tcp SSL config but where is that?

I tried to configure my browser to allow self-signed and I failed, it's getting harder with every day getting any buggy-SSL work on a browser.

Going a step back, does anyone have an lean/minimalistic example how I get my own SSL cert + file configured? (I can't use the LetsEncrypt way and I already have the files...)

Thanks!

Tilwen

**Comment by maxhelias** [comment, marked as answer]
Hi!
I think this should help you : https://github.com/dunglas/symfony-docker/blob/main/docs/tls.md
